pkgs/desktops/plasma-5/discover.nix

```diff
@@ -0,0 +1,15 @@
+{
+  mkDerivation,
+  extra-cmake-modules, gettext, kdoctools, python,
+  qtquickcontrols2,
+  karchive, kconfig, kcrash, kdbusaddons, kdeclarative, kio, kitemmodels, kxmlgui
+}:
+
+mkDerivation {
+  name = "discover";
+  nativeBuildInputs = [ extra-cmake-modules gettext kdoctools python ];
+  buildInputs = [
+    qtquickcontrols2
+    karchive kconfig kcrash kdbusaddons kdeclarative kio kitemmodels kxmlgui
+  ];
```

Comment:
meta, please!

Reply:
None of the software distributed as part of KDE Applications carry meta info.

Reply:
Oh? Interesting, well nevermind then :1+.